packages/material-ui-lab/src/YearPicker/PickersYear.tsx

```diff
@@ -15,90 +26,130 @@ export interface YearProps {
+};
+
+const PickersYearRoot = experimentalStyled(
+  'div',
+  {},
+  {
+    name: 'MuiPickersYear',
+    slot: 'Root',
+    overridesResolver: (
+      props: { styleProps: StyleProps },
+      styles: Partial<Record<PickersYearClassKey, object>>,
+    ) => {
+      const { styleProps } = props;
+      return {
+        ...styles.root,
+        ...(styleProps.wrapperVariant &&
+          styles[`mode${capitalize(styleProps.wrapperVariant)}` as PickersYearClassKey]),
+      };
+    },
   },
-  modeDesktop: {
+)(({ styleProps }) => ({
+  flexBasis: '33.3%',
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  ...(styleProps?.wrapperVariant === 'desktop' && {
     flexBasis: '25%',
+  }),
+}));
+
+const PickersYearButton = experimentalStyled(
+  'button' as const,
+  {},
+  {
+    name: 'MuiPickersYear',
+    slot: 'YearButton',
+    overridesResolver: () => ({}),
+  },
+)(({ theme }) => ({
+  color: 'unset',
+  backgroundColor: 'transparent',
+  border: 0,
+  outline: 0,
+  ...(theme.typography.subtitle1 as React.CSSProperties),
+  margin: '8px 0',
+  height: 36,
+  width: 72,
+  borderRadius: 16,
+  cursor: 'pointer',
+  '&:focus, &:hover': {
+    backgroundColor: alpha(theme.palette.action.active, theme.palette.action.hoverOpacity),
+  },
+  [`&.${pickersYearClasses.disabled}`]: {
+    color: theme.palette.text.secondary,
   },
-  yearButton: {
-    color: 'unset',
-    backgroundColor: 'transparent',
-    border: 0,
-    outline: 0,
-    ...theme.typography.subtitle1,
-    margin: '8px 0',
-    height: 36,
-    width: 72,
-    borderRadius: 16,
-    cursor: 'pointer',
+  [`&.${pickersYearClasses.selected}`]: {
+    color: theme.palette.primary.contrastText,
+    backgroundColor: theme.palette.primary.main,
     '&:focus, &:hover': {
-      backgroundColor: alpha(theme.palette.action.active, theme.palette.action.hoverOpacity),
-    },
-    '&$disabled': {
-      color: theme.palette.text.secondary,
-    },
-    '&$selected': {
-      color: theme.palette.primary.contrastText,
-      backgroundColor: theme.palette.primary.main,
-      '&:focus, &:hover': {
-        backgroundColor: theme.palette.primary.dark,
-      },
+      backgroundColor: theme.palette.primary.dark,
     },
   },
-  disabled: {},
-  selected: {},
-});
+}));
 
 /**
  * @ignore - internal component.
  */
-const PickersYear = React.forwardRef<HTMLButtonElement, YearProps & WithStyles<typeof styles>>(
-  (props, forwardedRef) => {
-    const { autoFocus, classes, children, disabled, onClick, onKeyDown, selected, value } = props;
-    const ref = React.useRef<HTMLButtonElement>(null);
-    const refHandle = useForkRef(ref, forwardedRef as React.Ref<HTMLButtonElement>);
-    const wrapperVariant = React.useContext(WrapperVariantContext);
+const PickersYear = React.forwardRef<HTMLButtonElement, YearProps>((inProps, forwardedRef) => {
+  // eslint-disable-next-line material-ui/mui-name-matches-component-name
```

Comment:
Using a named function instead of an arrow function should do it

```suggestion
```

Reply:
> Using a named function instead of an arrow function should do it

i.e.

`React.forwardRef(function Component() {})`